source/auth.rst

```diff
@@ -1,30 +1,94 @@
 .. toctree::
 
 .. _`FedCloud Check-in client`: https://aai.egi.eu/fedcloud
+.. _`AppDB`: https://appdb.egi.eu
 
 Authentication
 ==============
 
-.. _oidc-auth-using-check-in:
+Authorisation
+-------------
 
-OpenID Connect using Check-in
------------------------------
+Cloud Compute service is accessed through **Virtual Organisations (VOs)**. A VO
+is a grouping of IaaS cloud providers from the federation, who allocate capacity
+for a specific user group. Users with similar interest/requirements can join or
+form a VO to gather resources from EGI cloud providers - typically for a given
+project, experiment or use case. There are generic VOs too, for example the
+vo.access.egi.eu VO, which is open for any user who wants to experiment with the
+service. **You have to join a VO before you can interact with the cloud
+resources**, while higher level services (PaaS, SaaS) do not always require
+VO membership.
 
-`OpenID Connect <http://openid.net/connect/>`_ is an authentication protocol
-based on OAuth 2.0 and replacing the current VOMS-based authentication on the
-EGI Cloud.
+Users that are members of a VO will have access to the providers supporting
+that VO: they will be able to manage VMs, block storage and object storage
+available to the VO. Resources (VMs and storage) are shared across all members
+of the VO, please do not interfere with the VMs of other users if you are not
+entitled to do so (specially do not delete them).
 
-The providers natively integrated with EGI Check-in are able to authenticate
-users with OAuth 2.0 tokens without using any certificates in the process by
-relying on the OpenStack Keystone `OS-FEDERATION API <https://developer.openstack.org/api-ref/identity/v3-ext/index.html#os-federation-api>`_.
+Some users roles have special consideration in VOs:
 
-.. TODO(enolfc): discovery of providers with check-in support?
+* Users with VO Manager, VO Deputy or VO Expert Role have extra pivileges in
+  the AppDB for managing the Virtual Appliances to be avaialable at every
+  provider. Check the AppDB guide on VO-wide image list management for more
+  information.
 
+
+Pilot VO
+^^^^^^^^
+
+The `vo.access.egi.eu Virtual Organisation <https://operations-portal.egi.eu/vo/view/voname/vo.access.egi.eu>`_
+serves as a test ground for users to try the Cloud Compute service and to
+prototype and validate applications. It can be used for up to 6 month by any
+new user.
+
+.. warning::
+
+  * After the 6-month long membership in the fedcloud.egi.eu VO, you will need
+    to move to a production VO, or establish a new VO.
+
+  * The resources are not guaranteed and may be removed without notice by
+    providers. **Back-up frequently to avoid loosing your work!**
```

Comment:
Would be nice if we could provide some pointers for this. (or we can agree it's out of our scope)

Reply:
do we have any nice pointers?

Reply:
None that I can think off, but we could have a section/placeholder/draft mentioning things like using snapshots at the IaaS level and maybe something about what you used to backup the notebooks? At least we have some knowledge/experience with this.

Reply:
I have created an issue #17, to be added to the solutions or storage sections.

Reply:
Fair enough, thanks!